### PR TITLE
NavBar

### PR DESCRIPTION
**Pull request by shsieh005**
Screenshot 2024-04-14 at 3 41 14 PM

### CODE REVIEW

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+import React, { useState } from "react";
+import logo from "@/public/logo.webp";
+import { items } from "@/data/Navigation";
+
 const NavBar = () => {
+  const [selected, setSelected] = useState("");
   return (
-    <div className="fixed z-10 w-full bg-white bg-opacity-50">
-      <p>NavBar</p>
+    <div className="px-8 fixed z-30 h-[10vh] bg-white w-screen flex justify-between items-center text-xl">
```
Comment (seanquiambao) [inline src/components/NavBar.jsx:11]: make the text 2xl, and make the navbar a little bit thinner.

**Review by seanquiambao — CHANGES_REQUESTED** [review]
navigation bar looks great! just a few touch ups

**Comment by seanquiambao** [comment]
don't forget to link your issues!

**Comment by shsieh005** [comment]
image
I got this message in terminal after i tried to push the changes. I git pull origin to try to solve the conflict but it can't be resolved. How can I fix it?

**Comment by seanquiambao** [comment]
> image
> 
> I got this message in terminal after i tried to push the changes. I git pull origin to try to solve the conflict but it can't be resolved. How can I fix it?

try git pull --rebase

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**